kamon-cats-io/src/main/scala/kamon/instrumentation/catseffect/CatsIoInstrumentation.scala

```diff
@@ -0,0 +1,39 @@
+package kamon.instrumentation.catseffect
+import kamon.Kamon
+import kamon.context.{Context, Storage}
+import kamon.instrumentation.Mixin.HasContext
+import kanela.agent.api.instrumentation.mixin.Initializer
+import kanela.agent.libs.net.bytebuddy.asm.Advice
+import kanela.agent.scala.KanelaInstrumentation
+
+class CatsIoInstrumentation extends KanelaInstrumentation {
+
+  forTargetType("cats.effect.internals.IOShift.Tick") { builder =>
```

Comment:
hey @jatcwang awesome!!. The only thing you need for the instrumentation to work is:

change this: 
```scala 
  forTargetType("cats.effect.internals.IOShift.Tick") 
```
to 

```scala
forTargetType("cats.effect.internals.IOShift$Tick")
```

Reply:
Ah haha I tried IOShift$.Tick :pray: